src/inausoft.netCLI.Tests/LogicalCommandDeserializerTests.cs

```diff
@@ -39,11 +38,56 @@ public void RegexArgumentHandler_DeserializeProperCommand_ForMultipleOptions()
             Assert.AreEqual(intOptionValue, command.IntOption);
         }
 
+        [TestMethod]
+        public void LogicalCommandDeserializer_DeserializeProperCommand_ForMultipleOptions_UsingShortNames()
+        {
+            //Arrange
+            var stringOptionValue = "sampleString";
+            var intOptionValue = 102;
+
+            var args = new string[]
+                            {
+                                "-b",
+                                "-s", stringOptionValue,
+                                "-i", intOptionValue.ToString()
+                            };
+
+            //Act
+            var command = Deserializer.Deserialize<Command1>(args);
```

Comment:
Maybe is it me, but to understand the code I needed to go find what is inside Command1 class, maybe it will be good to give it a more descriptive name - just a suggestion

Reply:
Can you propose change?

Reply:
I understand that it is just some mocked command, nothing better to propose at the moment